packages/salesforcedx-vscode-apex/package.json

```diff
@@ -157,10 +157,6 @@
           "command": "sfdx.force.apex.test.run",
```

Comment:
You should be able to also remove `salesforce/salesforcedx-sobjects-faux-generator` as a dependency in this extension

Reply:
Sure, will do that.